ci/.gitlab-ci.squidex.yml

```diff
@@ -12,11 +12,9 @@ setup:squidex-image:
     - docker login -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD $CI_REGISTRY
     - docker build -t $SQUIDEX_SYNC_DOCKER_IMAGE . -f dev/base.Dockerfile
     - docker push $SQUIDEX_SYNC_DOCKER_IMAGE
-  only:
-    refs:
-      - master
-    changes:
-      - dev/base.Dockerfile
+  rules:
```

Comment:
whats going on here?

Reply:
I was trying to make the build only run when there's actual changes to those files. I don't think I was successful

Reply:
i see, and it didn't work the way it was previously done?

Reply:
Nope. This probably has something to doe with the fact that our CI is triggered via webhook (from github) and not from a push. https://gitlab.com/gitlab-org/gitlab/-/issues/27318#note_248733490